docs/COMMITTER_GUIDE.md

```diff
@@ -0,0 +1,41 @@
+<!--
+Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the GPL, Version 3.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.gnu.org/licenses/gpl-3.0.txt
+-->
+
+# Committer Guidelines
+
+These are the guidelines for people with commit privileges on the GitHub repository. Committers act as members of the Core Team and not necessarily employees of Dell.
+
+These guidelines apply to everyone and as Committers you have been given access to commit changes because you exhibit good judgment and have demonstrated your commitment to the vision of the project. We trust that you will use these privileges wisely and not abuse it.
+
+If these privileges are abused in any way and the quality of the project is compromised, our trust will be diminished and you may be asked to not commit or lose these privileges all together.
+
+## General Rules
+
+### Don't
+
+* Break the build.
+* Commit directly.
```

Comment:
I am wondering if this is going to be an acceptable condition for the Ansible Core team. They help us out occasionally by doing direct commits and have never broken anything this way.

Reply:
Whom do you mean by "Ansible core team" ? when we have common templates and process for OneAnsible its better to stick to this.

Reply:
They are Redhat employees who develop and maintain Ansible. They occasionally do small (typically one-line) changes in our repo that are things required for all Ansible collections to stay current with the currently available versions of Ansible software. An example would be the following commit/PR:  (Although a PR was opened for this, the submitter merged it himself almost immediately after opening it.)

https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/66

Reply:
Raising a PR is anyway mandatory. i.e. Users should not commit directly. Hence, we shall retain this point "commit directly" in the DON'Ts. In order to provide exceptions for RedHat people for such simple changes, we will add some extra description in the template. We shall do this later, after the technical team updates the word document that contains the templates.

Reply:
Based on our discussion for this point and the reasons you have stated here, it is acceptable to me to retain this item in the DON'Ts" list. I will mark this issue "Resolved".